sdk/communication/AzureCommunicationChat/Source/Generated/Models/MicrosoftTeamsUserIdentifierModel.swift

```diff
@@ -0,0 +1,67 @@
+// --------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// --------------------------------------------------------------------------
+
+import AzureCore
+import Foundation
+// swiftlint:disable superfluous_disable_command
+// swiftlint:disable identifier_name
+// swiftlint:disable line_length
+// swiftlint:disable cyclomatic_complexity
+
+/// A Microsoft Teams user.
+public struct MicrosoftTeamsUserIdentifierModel: Codable, Equatable {
```

Comment:
let's make sure the model types are not public

Reply:
The `public` was automatically added in the generated file. Should this be removed (we would need to manually do it though)

Reply:
In some other languages there are ways to inform the code generator to generate as non public. I am wondering if autorest.swift already has mechanisms for it.

Reply:
Had a quick chat with @tjprescott and looks like we don't have this mechanism for swift autorest